src/Fields/field.jl

```diff
@@ -125,3 +127,5 @@ ZFaceField(arch::AbstractArchitecture, grid, args...) = ZFaceField(eltype(grid),
 @propagate_inbounds Base.setindex!(f::Field, v, inds...) = @inbounds setindex!(f.data, v, inds...)
 
 gpufriendly(f::Field) = data(f)
+
+Adapt.adapt_structure(to, field::Field) = Adapt.adapt(to, averaged_field.data)
```

Comment:
```suggestion
Adapt.adapt_structure(to, field::Field) = Adapt.adapt(to, field.data)
```
Not sure if typo?

Reply:
Thanks for catching this --- I didn't mean to add! However, this change may be needed / helpful for getting the `Simulation` tests to pass in an upcoming PR. It would also close #746 . I'm starting to think its a good idea. It makes life pretty simple.